torchmetrics/image/ssim.py

```diff
@@ -57,8 +57,8 @@ class StructuralSimilarityIndexMeasure(Metric):
 
     def __init__(
         self,
-        kernel_size: Sequence[int] = (11, 11),
-        sigma: Sequence[float] = (1.5, 1.5),
+        kernel_size: Union[int, Sequence[int]] = 11,
+        sigma: Union[float, Sequence[float]] = 1.5,
```

Comment:
I'd update types and default values for `kernel_size` and `sigma` accordingly for functional metrics as well.

Reply:
Thanks for the update! Sorry I missed this here...

Reply:
I updated the same thing in the functional file